packages/react-icon-provider/README.md

```diff
@@ -0,0 +1,25 @@
+# @fluentui/react-icon-provider
+
+**Icon override provider for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
+
+This component is used for providing overrides to the icons in `@fluentui/react-icons-mdl2` and `@fluentui/react-icons`.
```

Comment:
Can you clarify that the provider doesn't affect legacy icons or their overrides? (`initializeIcons` or `registerIcons` from `@fluentui/font-icons-mdl2`)